libs/csm/rest/csm_rest_query_deployment.py

```diff
@@ -216,3 +206,30 @@ def verify_storage_set(self, cluster_id: str, storage_set_id: str = None,
             self.log.error(err_msg)
             result = False
         return resp, result, err_msg
+
+    def verify_error_message(self, resp, resp_error_code, resp_msg_id, resp_msg_index):
+        """
+        Verify error details
+        """
+        result = True
+        if CSM_REST_CFG["msg_check"] == "enable":
+            resp_data = self.rest_resp_conf[resp_error_code][resp_msg_id]
+            msg = resp_data[resp_msg_index]
+            if resp.json()["error_code"] != resp_error_code:
+                self.log.error("Error code check failed")
+                result = False
+            else:
+                self.log.info("Error code check passed")
+            if resp.json()["message_id"] != resp_msg_id:
+                self.log.error("Message id check failed")
+                result = False
+            else:
+                self.log.info("Message id check passed")
+            if resp.json()["message"] != msg:
+                self.log.error("Message check failed")
+                result = False
+            else:
+                self.log.info("Message check passed")
+        else:
+            result = True
+        return result
```

Comment:
Can u return error message also?
combine all self.log.error in one variable and return it

Reply:
done